docs/reference/usage.rst

```diff
@@ -77,10 +76,6 @@ logger consumer creation::
         {
             $message = $event->getMessage();
 
-            if (!in_array($message->getValue('level'), $this->types)) {
-                throw new InvalidParameterException();
```

Comment:
I've read somewhere that the replacement would be `\RuntimeException`, so maybe switch to that?

Reply:
Also, providing a message would be great.

Reply:
Thanks.
Sorry I've messed up rebase, here is the new PR https://github.com/sonata-project/SonataNotificationBundle/pull/376